### PR TITLE
Validation Error

### LINKED ISSUE #76: Add form validation - Publisher interface
Warn a user if:
* DOI is malformed
* email address not valid / not separated by newline

Related to #22

### PR DESCRIPTION
**Pull request by jinankjain**
Fixes #76 .

@acabunoc I have added validation test for DOI and Email ID.

### CODE REVIEW

File: templates/pages/paper.jsx [hunk 1]
```diff
@@ -20,11 +20,31 @@ var Issue = React.createClass({
       this.replaceWith('home');
     }
   },
+  validateEmail: function(emailId) {
+    var emailRe = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```
Comment (acabunoc) [inline templates/pages/paper.jsx]: Thanks @jinankjain! I'm not sure if a regex is the best way to handle this -- have you looked at any packages? 

This seems interesting:
https://www.npmjs.com/package/validator

**Comment by jinankjain** [comment]
@acabunoc I have made the necessary changes :smile:

**Comment by acabunoc** [comment]
Thanks @jinankjain - I'll test this out tomorrow!

**Comment by acabunoc** [comment]
This is great! Thanks @jinankjain r+

👏 💯 👍 

Sorry for the delay!

**Pull request merged**